f2/src/forms/ConfirmationForm.js

```diff
@@ -42,30 +42,31 @@ export const ConfirmationForm = (props) => {
           <Trans id="analystReport.selfHarmWord" />
         </div>
       ) : null}
-      <Form
-        onSubmit={props.onSubmit}
-        render={({ handleSubmit }) => (
-          <Stack
-            as="form"
-            onSubmit={handleSubmit}
-            shouldWrapChildren
-            spacing={6}
-          >
+      <Formik
+        initialValues={{}}
```

Comment:
Is initialValues necessary here?

Reply:
Yeah. This looks odd, but if I remove this code, I got "TypeError: Cannot convert undefined or null to object"